weave-js/src/react.tsx

```diff
@@ -599,71 +620,74 @@ export const parseRef = (ref: string): ObjectRef => {
     };
   }
 
-  if (isWeaveRef) {
-    const trimmed = trimStartChar(decodedUri, '/');
-    const tableMatch = trimmed.match(RE_WEAVE_TABLE_REF_PATHNAME);
-    if (tableMatch !== null) {
-      const [entity, project, digest] = tableMatch.slice(1);
-      return {
-        scheme: 'weave',
-        entityName: entity,
-        projectName: project,
-        weaveKind: 'table' as WeaveKind,
-        artifactName: '',
-        artifactVersion: digest,
-        artifactRefExtra: '',
-      };
-    }
-    const callMatch = trimmed.match(RE_WEAVE_CALL_REF_PATHNAME);
-    if (callMatch !== null) {
-      const [entity, project, callId] = callMatch.slice(1);
-      return {
-        scheme: 'weave',
-        entityName: entity,
-        projectName: project,
-        weaveKind: 'call' as WeaveKind,
-        artifactName: callId,
-        artifactVersion: '',
-        artifactRefExtra: '',
-      };
-    }
-    const match = trimmed.match(RE_WEAVE_OBJECT_REF_PATHNAME);
-    if (match === null) {
-      throw new Error('Invalid weave ref uri: ' + ref);
-    }
-    const [
-      entityName,
-      projectName,
-      weaveKind,
-      artifactName,
-      artifactVersion,
-      artifactRefExtra,
-    ] = match.slice(1);
+  throw new Error(`Unknown protocol: ${url.protocol}`);
+};
+
+/**
+ * This is the "New Weave" ref parser that doesn't need to worry about W&B
+ * artifacts or local artifacts. It is not exported currently, but is used for
+ * the new Weave branch in parseRef
+ */
+const parseWeaveTraceRef = (ref: string): WeaveObjectRef => {
```

Comment:
This was extracted from `parseRef` and is easier to review with whitespace hidden